JetNews/app/src/main/java/com/example/jetnews/ui/home/PostCards.kt

```diff
@@ -128,32 +133,75 @@ fun PostCardSimple(
 
 @Composable
 fun PostCardHistory(post: Post, navigateToArticle: (String) -> Unit) {
+    var openDialog by remember { mutableStateOf(false) }
     Row(
-        Modifier
-            .clickable(onClick = { navigateToArticle(post.id) })
-            .padding(16.dp)
+        Modifier.padding(16.dp)
     ) {
-        PostImage(
-            post = post,
-            modifier = Modifier.padding(end = 16.dp)
-        )
-        Column(Modifier.weight(1f)) {
+        Row(
+            Modifier
+                .clickable(onClick = { navigateToArticle(post.id) })
+                .weight(1f)
+        ) {
+            PostImage(
+                post = post,
+                modifier = Modifier.padding(end = 16.dp)
+            )
+            Column(Modifier.weight(1f)) {
+                CompositionLocalProvider(LocalContentAlpha provides ContentAlpha.medium) {
+                    Text(
+                        text = "BASED ON YOUR HISTORY",
+                        style = MaterialTheme.typography.overline
+                    )
+                }
+                PostTitle(post = post)
+                AuthorAndReadTime(
+                    post = post,
+                    modifier = Modifier.padding(top = 4.dp)
+                )
+            }
+        }
+        Row() {
```

Comment:
Using two Rows inside another Row might not be the best way to go. Consider making the Icon 48x48 (with padding) and just put it inside the top level row together with the rest. The inner Clickable should override the outer one, right?

Reply:
You are right!